.github/workflows/release.yml

```diff
@@ -8,9 +8,13 @@ permissions:
   contents: read # for checkout
 
 jobs:
+  test:
+    uses: "./.github/workflows/node.js.yml"
+
```

Comment:
We declare tests just once in CI instead of duplicating them like we used to. Here's an example actions run of what that should look like on [main](https://github.com/defenseunicorns/kubernetes-fluent-client/actions/runs/11130649516/job/30930487410).